Sources/ProjLintKit/Rules/FileExistenceOptions.swift

```diff
@@ -3,19 +3,12 @@ import Foundation
 class FileExistenceOptions: RuleOptions {
     let existingPaths: [String]?
     let nonExistingPaths: [String]?
+    let allowedPathsRegex: [String]?
 
     override init(_ optionsDict: [String: Any], rule: Rule.Type) {
-        let existingPaths = RuleOptions.optionalStringArray(forOption: "existing_paths", in: optionsDict, rule: rule)
-        let nonExistingPaths = RuleOptions.optionalStringArray(forOption: "non_existing_paths", in: optionsDict, rule: rule)
-
-        guard existingPaths != nil || nonExistingPaths != nil else {
```

Comment:
Why was this check removed? I think this is intended to ensure this rule isn't misconfigured.

Reply:
Ups. I brought it back